src/chat/UnreadNotice.js

```diff
@@ -50,22 +50,24 @@ class UnreadNotice extends PureComponent<Props> {
 
   render() {
     const { limited, narrow, unreadCount } = this.props;
-
+    const shouldShow = unreadCount > 0;
```

Comment:
I don't think this is adding anything to the code. If it was a complicated piece of code, sure.